.gitignore

```diff
@@ -1,3 +1,38 @@
 # OSX
-#
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Build
+*.aar
```

Comment:
we don't want this gitignored

Reply:
there are fine grained gitignores in the individual directories

Reply:
you could for instance gitignore this in the `gomobile` (previously called `sdk` directory)

Reply:
why would we want to commit binaries?